.openshift/templates/jenkins-ephemeral.yml

```diff
@@ -0,0 +1,241 @@
+apiVersion: template.openshift.io/v1
```

Comment:
why carry this vs use the one embedded in the cluster?

Reply:
We are mounting the global library xml ConfigMap